arrebol/worker/manager/manager.go

```diff
@@ -0,0 +1,34 @@
+package manager
+
+import (
+	"github.com/ufcg-lsd/arrebol-pb/arrebol/worker"
+	"github.com/ufcg-lsd/arrebol-pb/storage"
+)
+
+type Manager struct {
+	storage *storage.Storage
+}
+
+func NewManager(storage *storage.Storage) *Manager {
+	return &Manager{
+		storage:storage,
+	}}
```

Comment:
out of curiosity, the formatting of this NewManager func is ok?

Reply:
The syntax is ok but the indent was not. Fixed.